frontend/src/http-common.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+export default axios.create({
+  baseURL: "http://localhost:8080/api",
+  headers: {
+    "Content-type": "application/json",
+    "Access-Control-Allow-Origin":"*"
```

Comment:
I do not think this header ""Access-Control-Allow-Origin"" will be needed if we fix cors on backend side.